### PR TITLE
Fix cargo build directory to libindy

### PR DESCRIPTION
**Pull request by sklump**
Build from where cargo build can find Cargo.toml

### CODE REVIEW

File: doc/ubuntu-build.md [hunk 1]
```diff
@@ -16,9 +16,10 @@
 1. Checkout and build the library:
    
    ```
-   git checkout https://github.com/hyperledger/indy-sdk.git
-   cd ./indy-sdk
+   git clone https://github.com/hyperledger/indy-sdk.git
+   cd ./indy-sdk/indy-sdk
```
Comment (jovfer) [inline doc/ubuntu-build.md]: Should it be `./indy-sdk/libindy/` ?

Comment (vimmerru) [inline doc/ubuntu-build.md]: Should it be  ./indy-sdk/libindy ???

Comment (sklump) [inline doc/ubuntu-build.md]: Yes, I am sure you are correct.

Comment (sklump) [inline doc/ubuntu-build.md]: Yes, thanks. My mistake.

File: doc/ubuntu-build.md [hunk 2]
```diff
@@ -19,9 +19,10 @@
    git clone https://github.com/hyperledger/indy-sdk.git
    cd ./indy-sdk/libindy
    cargo build
+   cd ..
    ```
 1. Run integration tests:
-   * Start local nodes pool on `127.0.0.1:9701-9708` with Docker:
+   * Start local nodes pool on `127.0.0.1:9701-9709` with Docker:
```
Comment (jovfer) [inline doc/ubuntu-build.md]: Actually pool use only 9701-9708 ports. So this change should not be applied.
@vimmerru I think we should fix it in appropriate dockerfile also. May be we can merge this PR, and fix this line and dockerfile in the new one.

**Pull request merged**